config/pipelines/high_throughput_lcm_triomics.wip.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Pipeline configuration for EMSeq
+LCM Triomics EMSeq:
+  pipeline_group: LCM Triomics WGS and EMSeq
```

Comment:
Strictly speaking, LCM Triomics is 3 things:
- RNA (we will add this later)
- DNA WGS
- DNA EMSeq

So maybe the group name is just LCM Triomics?

Reply:
`pipeline_group: LCM Triomics WGS and EMSeq` is for grouping multiple parts of the pipeline config together. In EMSeq branch of the pipeline, we have two separate sections of the pipeline config:

- LCM Triomics EMSeq
- LCM Triomics EMSeq Custom Pooling

Using the same group name links them in Limber "Work In Progress" and "Pipelines Overview" pages under LCM Triomics WGS and EMSeq, which is actually a single pipeline at the moment.